.drone.yml

```diff
@@ -5,4 +5,4 @@ build:
     - pip install -r requirements.txt
     - pip install -r test-requirements.txt
     - pip install python-coveralls
-    - make run-tests
+    - lettuce ensime_shared/spec/features
```

Comment:
Rather than change this from using the `make` target and leaving to two places where the test procedure has to be updated and maintained, it would make more sense to me to make a few adjustments to the `Makefile` and otherwise leave the CI config running `make run-tests`:

1. Make the `run-tests` target depend on `venv` one.
2. Make the `venv` target additionally depend on `$(ACTIVATE)`, so that it triggers creation of the virtualenv if it doesn't exist yet.
3. Remove the `pip install -r *.txt` steps here in the CI config as they're made redundant.

I think that'll do it, but haven't actually tried yet so I might be overlooking something.